lisa/tests/base.py

```diff
@@ -548,6 +548,9 @@ class RTATestBundle(TestBundle, metaclass=RTATestBundleMeta):
         0 : 100,
         # Feeble boards like Juno/TC2 spend a while in sugov
         r"^sugov:\d+$" : 5,
+        # The mailbox controller (MHU), now threaded, creates work that sometimes
+        # exceeds the 1% threshold.
+        r"^irq\/\d+-mhu_link$": 1.5
```

Comment:
Is escaping the forward-slash necessary ? It looks like this matches just as well:
```
import re
re.match(r"^irq/\d+-mhu_link$", 'irq/0-mhu_link')
```

Reply:
No, it seems it's not necessary. Thanks!